gp_fit_real_data/fit_uv_real_data.py

```diff
@@ -73,7 +66,7 @@ def objective_closure():
         m = gpflow.models.GPR(data=(time, uv_band_flux),
                               mean_function=Constant(np.mean(uv_band_flux)),
                               kernel=k,
-                              noise_variance=1)
+                              noise_variance=0.1)
```

Comment:
Not sure if this is necessary but assuming you've got a reasonable signal-to-noise ratio and normalized the Y to unit-variance, you probably want to initialize noise variance to something smaller so your fit doesn't end up being just a straight line explaining everything away as noise

Reply:
Hi T, thanks very much for taking a look at this! I take it that in the fixed noise case the value of the noise_variance parameter of the GPR class gets overwritten when m.likelihood.variance is assigned later on? In cases where the noise is optimised I'm curious to know what your thoughts are on Iain Murray's advice to set the initial noise level "high" on slide 34 of the following presentation: https://homepages.inf.ed.ac.uk/imurray2/teaching/08gp_slides.pdf Would you recommend any rule of thumb for determining what a high initial noise variance might be if you have a rough idea what the SNR is going to be?

Reply:
That's a fair point. There's generally at least TWO local minima - the "long lengthscales/small kernel variance, near-linear, high noise variance" and "short lengthscales/large kernel variance, wiggly, low noise variance" regimes. Depending on your data, one or the other may be overall more appropriate. Iain Murray is right that with high noise variance it'll be easier to vary the other parameters, but you're also more likely to end up in the high noise variance basin in the end. Random restarts - multiple trials with different initializations - is a good way forward!

Reply:
Makes sense, thanks!